bot/exts/info/doc/_redis_cache.py

```diff
@@ -29,13 +31,20 @@ async def set(self, item: DocItem, value: str) -> None:
         needs_expire = False
 
         with await self._get_pool_connection() as connection:
-            if redis_key not in self._set_expires:
+            set_expire = self._set_expires.get(redis_key)
+            if set_expire is None:
                 # An expire is only set if the key didn't exist before.
                 # If this is the first time setting values for this key check if it exists and add it to
                 # `_set_expires` to prevent redundant checks for subsequent uses with items from the same page.
```

Comment:
This comment doesn't make sense here any more

Reply:
Why not? The bock will execute on the first set of the key like the comment says

Reply:
The top line is fine. Just the next two about `_set_expires` are not relevant specifically to this block any more, as now `_set_expires` is always reset regardless of whether it already existed.